lib/proximity.js

```diff
@@ -401,9 +401,20 @@ function Proximity(opts) {
   var controller = null;
   var state = {};
   var raw = 0;
-  var freq = opts.freq || 25;
   var last = 0;
+  var rawFiltered = 0;
+  var lastFiltered = 0;
+  var freq = opts.freq || 25;
   var pinValue = typeof opts === "object" ? opts.pin : opts;
+  var isFiltered = opts.filter;
+  var filterDefaults = {
+    samples: 5,
+    soften: 2,
+    deviation: 2
+  };
+  var filterOpts = typeof opts.filter === "object" ? Object.assign(filterDefaults, opts.filter) : filterDefaults;
+
+  this.samples = [];
```

Comment:
Just that I thought some other classes might have `samples` properties, but I can't find any that do.